Makefile

```diff
@@ -9,7 +9,7 @@ all: bin/cfssl bin/cfssl-bundle bin/cfssl-certinfo bin/cfssl-newkey bin/cfssl-sc
 
 bin/%: $(shell find . -type f -name '*.go')
 	@mkdir -p $(dir $@)
-	go build -ldflags $(LDFLAGS) -o $@ ./cmd/$(@F)
+	GOOS=$(TARGETOS) GOARCH=$(TARGETARCH) go build -ldflags $(LDFLAGS) -o $@ ./cmd/$(@F)
```

Comment:
Is this makefile only to be used inside the Dockerfile? If not, then it probably should not contain the `TARGETOS` and `TARGETARCH` variables here (as they won't be present in other situations)

Reply:
Inside the Dockerfile, we are git cloning the `cfssl_trust` and executing its Makefile. This made the things tricky as I didn't wanted to touch `cfssl_trust` repository. This is a workaround solution that I could come up with for the issue. I have tested and can confirm that absence of values in `TARGETOS` and `TARGETARCH` variables does not cause any issues and `go build` command works fine.